spitfire/compiler/parser_test.py

```diff
@@ -86,7 +87,11 @@ def test_bad_do_syntax(self):
 #do #do $bar()
 #end def
     """
+    # Temporarily disable logging to avoid spurious exception output.
+    logging.disable(logging.ERROR)
     self.assertRaises(yappsrt.FatalParseError, self._compile, code)
```

Comment:
I guess this would be a problem whenever the `logging` module is used.

This case is `logging.exception`:
https://github.com/nicksay/spitfire/search?utf8=%E2%9C%93&q=%22logging.exception%22&type=Code

Here are the others:
`logging.error`
https://github.com/nicksay/spitfire/search?utf8=%E2%9C%93&q=%22logging.error%22&type=Code
`logging.warning`
https://github.com/nicksay/spitfire/search?utf8=%E2%9C%93&q=%22logging.warning%22&type=Code